scripts/rtma3d/exrtma3d_rhist.ksh

```diff
@@ -131,75 +131,66 @@ ln -sf  $COMOUThrrrdas_rtma3dm2 $DATA/$PDYm2
 ln -sf  $COMOUThrrrdas_rtma3dm3 $DATA/$PDYm3
 
 
-ls -1r */*/* | awk '
-
-            /prepbufr/ { print "./"$0 > "perm" ; next }
-
-	    /bufr/ { print "./"$0 > "perm" ; next }
+ls -1r */* | awk '
 
-            /grib2/ { print "./"$0 > "perm" ; next }
+            /rejectlist/ { print "./"$0 > "perm1" ; next }
 
-	    /grb2/ { print "./"$0 > "perm" ; next }
+            /smarthrrrconus/ { print "./"$0 > "perm1" ; next }
 
-            /rejectlist/ { print "./"$0 > "perm" ; next }
+	    /smarthrrrak/ { print "./"$0 > "perm1" ; next }
 
-	    /NASALaRCCloudInGSI/ { print "./"$0 > "perm" ; next }
+            /prslev/ { print "./"$0 > "perm1" ; next }
 
-            /NSSLRefInGSI/ { print "./"$0 > "perm" ; next }
+            /natlev/ { print "./"$0 > "perm1" ; next }
 
-            /LightningInGSI.bufr/ { print "./"$0 > "perm" ; next }
+            /howv/ { print "./"$0 > "perm1" ; next }
 
-            /satwnd/ { print "./"$0 > "perm" ; next }
+            /gust/ { print "./"$0 > "perm1" ; next }
 
-	    /nexrad/ { print "./"$0 > "perm" ; next }
+            /smartak/ { print "./"$0 > "perm1" ; next }
 
-	    /firstguess.nc/ { print "./"$0 > "perm" ; next }
-           
-            /subhprs/ { print "./"$0 > "perm" ; next } 
+            /wavebg/ { print "./"$0 > "perm1" ; next }
```

Comment:
At this point in time, we do not have any /wavebg/ files.  So you can remove from the script.